### ISSUE TITLE
Compilation Error for a typo

### ISSUE THREAD

**Issue by ragar90** [braintree/braintree_ios#374]
### General information

* SDK/Library version: 4.7.3
* Environment: Irelevant
* iOS Version and Device: All of them

### Issue description

Every time i do a pod install i got this same compilation error
```
/Users/ragar90/react-apps/givesafe_rn/ios/Pods/Braintree/BraintreeCore/BTAnalyticsMetadata.m:1:9: Non-portable path to file '"BTAnalyticsMetadata.h"'; specified path differs in case from file name on disk
```
The fix is quite simple change line from `#import "BTAnalyticsMetaData.h"` to `#import "BTAnalyticsMetadata.h"` i have to do it everytime and to be honest it's becoming tedious i don't know if this is problem with the version i have but it will be great if you can fix it.

**Comment by ragar90** [comment]
![screen shot 2017-11-29 at 7 04 21 pm](https://user-images.githubusercontent.com/1312458/33407670-0a18e862-d538-11e7-9263-c7f04a025d42.png)
<img width="1280" alt="screen shot 2017-11-29 at 11 02 50 am" src="https://user-images.githubusercontent.com/1312458/33407674-0b8e96e2-d538-11e7-99cd-f69f87454934.png">

**Comment by ragar90** [comment]
![screen shot 2017-11-29 at 7 04 21 pm](https://user-images.githubusercontent.com/1312458/33407679-15e9f8c0-d538-11e7-8aeb-752504b1e37a.png)
![screen shot 2017-11-29 at 7 04 12 pm](https://user-images.githubusercontent.com/1312458/33407680-15fe42c6-d538-11e7-83c0-1c3099153b9f.png)

**Comment by skunkworks** [comment]
Can you try updating to a newer version of the Braintree SDK? This should be fixed in later versions.

**Comment by demerino** [comment]
As suggested, this was fixed in `4.7.5`: https://github.com/braintree/braintree_ios/blob/master/CHANGELOG.md#475-2017-02-22
